index.html

```diff
@@ -2553,9 +2553,17 @@ <h3>Hypermedia-driven</h3>
         <p>
           <span class="rfc2119-assertion" id="arch-hypermedia">
             Interaction Affordances MUST include one or more Protocol Bindings.</span>
-          <span class="rfc2119-assertion" id="arch-hypermedia-protocol-binding">
-            <a>Protocol Bindings</a> MUST be serialized as <a href="#sec-hypermedia-controls">hypermedia controls</a> to be self-descriptive on how to
-            activate the <a>Interaction Affordance</a>.</span>
+
+          <span class="rfc2119-assertion" id="arch-hypermedia-protocol-binding-1">
+	    <a>Protocol Bindings</a> MUST be serialized in such a way that they are self-descriptive or otherwise 
+            clearly associated with definitions that indicate how to activate the <a>Interaction Affordance</a>.
+	  </span>
+
+          <span class="rfc2119-assertion" id="arch-hypermedia-protocol-binding-2">
+	    This MAY be achieved by <a href="#sec-hypermedia-controls">hypermedia controls</a>, 
+		by using <a>Profiles</a> or by using protocol specific metadata. 
```

Comment:
When the profiles is not published, we cannot really refer to it right? This is not a normative reference (not using [[my ref]] so it might be ok but it is still NOT clear what a profile can do. If we look at https://w3c.github.io/wot-architecture/#profile-description-methodology , it only says that it can constrain the protocol verbs etc. Thus, a profile cannot define a protocol binding by this definition.

Reply:
I think it's fine as long as there is a Profile definition. Using <a>Profiles</a> would expect this otherwise it will emphasize with a curled red line.

Reply:
Since WoT Profile 1.0 will now be published after WoT Architecture 1.1 and WoT Thing Description 1.1, I have proposed moving the definition of "profile" from the Profile specification to the TD or Architecture specification in order to resolve the dangling reference - see https://github.com/w3c/wot-thing-description/issues/1719.

Also see https://github.com/w3c/wot-thing-description/issues/1674 in which I proposed an updated description of protocol bindings in the TD specification which takes profiles into account. The wording in that issue describes the two different types of protocol bindings and how they are used.

It would be simpler if there were no normative definitions in the WoT Architecture specification at all, but as long as they exist they need to be kept in sync with the other specifications, since they all cross reference each other.